app/services/importer/import_beaches.rb

```diff
@@ -0,0 +1,126 @@
+module Importer
+  class ImportBeaches
+    include HTTParty
+
+    base_uri 'infopraiaapi.apambiente.pt'
+
+    def all
+      cleanup
+      import_beaches_deco
+      import_beaches_apa
+      Rails.logger.info "#{Store.where(store_type: :beach).count} total stores"
+    end
+
+    def cleanup
+      Store.where(store_type: :beach).delete_all
+    end
+
+    def import_beaches_apa
+      data = self.class.get('/perfil.json')&.body
+      JSON.parse(data)['features'].each do |item|
+        beach = item['attributes']
+        store = Store.find_or_initialize_by(name: beach['praia'], store_type: :beach)
```

Comment:
@simaob Shouldn't we just update the beaches?

Reply:
that's why we are filtering by `store_type: :beach`